packages/frontend/hooks/src/use-workspace-blob.ts

```diff
@@ -0,0 +1,40 @@
+import { workspaceManagerAtom } from '@affine/workspace/atom';
+import type { WorkspaceMetadata } from '@affine/workspace/metadata';
+import { useAtomValue } from 'jotai';
+import { useEffect, useState } from 'react';
+
+export function useWorkspaceBlobObjectUrl(
```

Comment:
I tried adding a lru cache for the blob, but the avatar will still flicker briefly because we create a new object url every time, but object url is hard to cache...

I think I'll leave this issue and resolve it in the future.

Reply:
Yeah, maybe not relevant to the workspace manger PR. Can revisit it later.